pkg/ephemeral-disk-utils/utils.go

```diff
@@ -66,13 +65,13 @@ func (om *OwnershipManager) SetFileOwnership(file string) error {
 	return os.Chown(file, uid, gid)
 }
 
-func RemoveFile(path string) error {
-	err := os.RemoveAll(path)
-	if err != nil && os.IsNotExist(err) {
-		return nil
-	} else if err != nil {
-		log.Log.Reason(err).Errorf("failed to remove %s", path)
-		return err
+func RemoveFilesIfExist(paths ...string) error {
+	var err error
+	for _, path := range paths {
+		err = os.Remove(path)
+		if !os.IsNotExist(err) {
```

Comment:
I think you need `if err != nil && !os.IsNotExist(err)`

Reply:
oh dear. I've added a test that should have caught that.